docs/configuration.md

```diff
@@ -982,6 +982,34 @@ Apart from these, the following properties are also available, and may be useful
     This is only relevant for the Spark shell.
   </td>
 </tr>
+<tr>
+  <td><code>spark.rpc.numRetries</code></td>
+  <td>3</td>
+    How many times for an RPC ask operation to retry before giving up.
```

Comment:
Number of times to retry before an RPC task gives up.

(We should also indicate whether 1 = retry once, or 1 = run it once in total)